<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="button">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_new">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_save">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_voxel">
        <property name="text">
         <string>Voxel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cvoxel">
        <property name="text">
         <string>Cut Voxel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_box">
        <property name="text">
         <string>Box</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cbox">
        <property name="text">
         <string>Cut Box</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_sphere">
        <property name="text">
         <string>Sphere</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_csphere">
        <property name="text">
         <string>Cut Sphere</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_ellipsoid">
        <property name="text">
         <string>Ellipsoid</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cellipsoid">
        <property name="text">
         <string>Cut Ellipsoid</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="75,25">
      <item>
       <layout class="QHBoxLayout" name="WIDGETGRID" stretch="90,10">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="GridSculptor" name="gridsculptor" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLCDNumber" name="lcdNumber"/>
          </item>
          <item>
           <widget class="QSlider" name="barraZ">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="50,50">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="20,80">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>R</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="SliderR">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="20,80">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>G</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="SliderG">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="20,80">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>B</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="SliderB">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,80">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>A</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="SliderA">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>200</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="Box">
           <attribute name="title">
            <string>Box</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>10</y>
              <width>121</width>
              <height>107</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="25,25,25,25">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Size Box</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="10,0">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineBoxX"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineBoxY"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Z</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineBoxZ"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="Sphere">
           <attribute name="title">
            <string>Sphere</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>121</width>
              <height>43</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4" stretch="50,50">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Radius Sphere</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="20,80">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Radius</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineRadius"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="Ellipsoid">
           <attribute name="title">
            <string>Ellipsoid</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>121</width>
              <height>101</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,25,25,25">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Radius Ellipsoid</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="20,80">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>RX</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineRadiusEllipX"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="20,80">
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>RY</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineRadiusEllipY"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="20,80">
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>RZ</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineRadiusEllipZ"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewsculp">
   <property name="text">
    <string>Newsculp</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GridSculptor</class>
   <extends>QWidget</extends>
   <header>gridsculptor.h</header>
   <container>1</container>
   <slots>
    <signal>updatePos(int,int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_new</sender>
   <signal>clicked(bool)</signal>
   <receiver>actionNewsculp</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gridsculptor</sender>
   <signal>updatePos(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>drawObject(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>barraZ</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>setPlanoZ()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderR</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>updatecolor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>824</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderG</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>updatecolor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>794</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>784</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderB</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>updatecolor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>814</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>782</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_voxel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setVOXEL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cvoxel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setCVOXEL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_box</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setBOX()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cbox</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setCBOX()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_sphere</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setSPHERE()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_csphere</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setCSPHERE()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>708</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ellipsoid</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setELLIPSOID()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>783</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cellipsoid</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setCELLIPSOID()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>881</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>772</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_save</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Salvar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>barraZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>678</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>drawObject(int,int)</slot>
  <slot>setPlanoZ()</slot>
  <slot>updatecolor()</slot>
  <slot>setVOXEL()</slot>
  <slot>setCVOXEL()</slot>
  <slot>setBOX()</slot>
  <slot>setCBOX()</slot>
  <slot>setSPHERE()</slot>
  <slot>setCSPHERE()</slot>
  <slot>setELLIPSOID()</slot>
  <slot>setCELLIPSOID()</slot>
  <slot>Salvar()</slot>
 </slots>
</ui>
